sdk/forms/src/protected-view/iframe/components/view.ts

```diff
@@ -26,16 +26,37 @@ type DisplayValueProps = {
   className?: string;
 };
 
-const DisplayValue = component(
-  ({ value: rootValue, label, path, className, clickToCopy, sendToParent }: DisplayValueProps): Node => {
-    const container = document.createElement('div');
+const DisplayValue = () =>
+  component(({ value: rootValue, label, path, className, clickToCopy, sendToParent }: DisplayValueProps) => {
```

Comment:
Why changing it to a function that return component?

Reply:
@ori-shalom Before this change, we always used the same instance of the DisplayValue component inside the map cycle above. This was treated like a prop change, though I was expecting each DisplayValue instance to be a new component instance. This caused a bug, when all DisplayValue except the last one got their event listeners removed. This is why I made this change.
The alternative solution here could be move render function from the `component` into a separate named const/function. Then get rid off DisplayValue component completely and call `component` with the extracted render function as an argument inside the map cycle directly (line 13). This should do the same thing